rules.md

```diff
@@ -354,17 +354,17 @@ Your claim will then be processed by an admin.
 
 ### 2.3.5 Processing claims
 
-An Admin will process new claims, assessing them based on the rules for the type of output you’ve produced and whether the work completes the Task adequately as it was defined.
+All claims are processed by Admins.  Claims are assessed based on the rules for the type of output produced, and whether the work completes the Task adequately as it was defined.  
 
 If the Admin approves the claim, they’ll leave a [Claim Approved comment](#435-task-claims) on the Trello Card, otherwise they may discuss any issues directly on the card comments with you.  Approved claims will be awarded usually within 7 days and will be listed in the [Claim Awards List](#).
 
 ## 2.5 Price List
 
 Each type of reward for Tasks in the App are listed below along with the min/max amounts of Dash that can be offered and basic criteria upon which claims will be approved.  
 
-Exact rewards are decided by the Bounty Admin with the price fixed or negotiated with a user who is reserving the relevant Tasks.
+Rewards are fixed, Dash-denominated amounts decided by Bounty Admins (typically the Bounty's Primary Admin).  Users may negotiate the amount with the appropriate Admin before requesting a reservation or making a claim.  For Tasks carried out by a Bounty's Primary Admin, the reward is decided by the Secondary Admin (see Section 5 for details). 
 
-Note that the Meta % is the commission added to the Task that Bounty Admins earn for creating and managing Tasks.
+The Meta % is an added commission that Admins earn for creating and managing Tasks.  For example, under 10% commission, an approved 10 DASH Task yields 10 DASH for the user completing the task, and 1 DASH for the Admin processing the claim.
```

Comment:
Ambiguity? Does this need to be clearer on which admin receives the rewards (rather than possibly both)

Reply:
Same issue as raised in other comments.  Hopefully the ambiguity is resolved if one reads the Admin section.  If not, there's a problem, if so, the ideal solution might be document restructuring.

Reply:
Yes, I agree, this is probably a matter for a future PR addressing some of the existing grammatical & other idiosyncrasies, rather than the subsatnce of your changes. We can refer back to it when addressing those at a later date.